src/sentry/integrations/jira/client.py

```diff
@@ -122,6 +123,11 @@ def create_comment(self, issue_key, comment):
     def update_comment(self, issue_key, comment_id, comment):
         return self.put(self.COMMENT_URL % (issue_key, comment_id), data={"body": comment})
 
+    def get_projects_paginated(self, params: dict[str, str | Any] | None = None):
+        response = self.get(self.PROJECTS_PAGINATED_URL, params=params)
+        return response
+
+    # deprecated - please use paginated one above
     def get_projects_list(self):
```

Comment:
I'd move this comment to a [docstring](https://peps.python.org/pep-0257/#one-line-docstrings), that way it'll show up in IDE tooltips.